assignments/r-6.md

```diff
@@ -6,4 +6,13 @@ language: R
 exercises: ['Making Choices 1', 'Making Choices 2', 'Strings 8', 'Making Choices 3']
 ---
 
+####Learning Objectives
+
+> Following this assignment students should be able to:
+
+> - understand and use the basic choice operators
```

Comment:
I'm guessing by "choice operators" you mean `==`, `>`, `<=`, etc., in which case the formal term (which we should use) is "relational operators".